pkg/status/status.go

```diff
@@ -35,9 +35,9 @@ func GetStatus() (map[string]interface{}, error) {
 	hostname, err := util.GetHostname()
 	if err != nil {
 		log.Errorf("Error grabbing hostname for status: %v", err)
-		stats["metadata"] = host.GetPayload("unknown")
+		stats["metadata"] = host.GetPayloadFromCache("unknown")
```

Comment:
This causes a lot of logs that are just incomprehensible. This will grab it from the cache instead of grabbing it directly.